server/api/api/endpoints/workflows.py

```diff
@@ -193,6 +193,7 @@ async def submit_workflow(
                 workflow_request=updated_request,
                 db_session=db_session,
                 auth_info=auth_info,
+                remote_debugging=workflow_request.spec.remote_debugging,
```

Comment:
I believe you can get it from the updated_request instead of adding another param